pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -132,3 +134,17 @@ def makevector(*x):
         return torch.tensor(x, dtype=torch_dtype)
 
     return makevector
+
+
+@pytorch_funcify.register(OpFromGraph)
+def pytorch_funcify_OpFromGraph(op, node=None, **kwargs):
+    _ = kwargs.pop("storage_map", None)
+
+    PYTORCH.optimizer(op.fgraph)
+    fgraph_fn = torch.compile(pytorch_funcify(op.fgraph, **kwargs))
```

Comment:
Do you need to compile the inner function? Is that a thing in PyTorch?

Reply:
I was following what numba does where it jits the inner function - we could remove the inner torch.compile and just return op.fgraph if that seems more reasonable. That will still lead to some c-linker issues fwiw.

Reply:
I removed the inner function, you only need to do indexing if the number of return values is more than 1

Reply:
Numba can only have inner compiled functions, I don't know if that's a requirement in pytorch, and whether it has any advantages. We don't do it for JAX

Reply:
I do not see / know of any requirement to have an inner compiled function.